src/main/java/gift/controller/WishlistController.java

```diff
@@ -31,10 +31,9 @@ public ResponseEntity<Page<WishlistItem>> getWishlist(@PathVariable("id") Long u
         return new ResponseEntity<>(wishlist, HttpStatus.OK);
     }
 
-
-    @PostMapping("/save")
-    public ResponseEntity<List<WishlistItem>> createWishlist(@RequestBody List<WishlistItem> wishlistItems) {
-        List<WishlistItem> savedItems = wishlistService.saveWishlistItems(wishlistItems);
+    @PostMapping("/{id}/save")
```

Comment:
POST 요청이 기본적으로 등록을 의미하기 때문에 굳이 save를 붙여줄 필요는 없을 것 같습니다~
```suggestion
    @PostMapping("/{id}")
```